ship-it-inc/ship-it-fend
# #168365508 Implement side bar for small screens

#### What does this PR do?
It implements sidebar nav for small screens
#### Description of Task to be completed?
- add implementation to the app
#### How should this be manually tested?
- Clone the project and cd into the branch
- start the app and login into the app
- you should see the sidebar on a small screen
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?

#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
